pkgs/development/python-modules/notebook/default.nix

```diff
@@ -17,28 +17,28 @@
 
 buildPythonPackage rec {
   pname = "notebook";
-  version = "7.2.0";
-  disabled = pythonOlder "3.8";
+  version = "7.2.2";
+  pyproject = true;
 
-  format = "pyproject";
+  disabled = pythonOlder "3.8";
```

Comment:
Lately, I have gone with removing those useless `disabled = pythonOlder "3.<old_version>"`.

Reply:
I don't mind if others delete it, but I don't actively delete it because I think it's a piece of meta-information.
Also, since this is a backport, there is no need to pick such a refactoring unless it is necessary.

Reply:
Definitely, this makes sense.